script.js

```diff
@@ -69,14 +85,60 @@ function shuffleCards(array) {
 let shuffledCardsArray = shuffleCards(cardObjectsArray);
 let cardDivArray = document.querySelectorAll('.card');
 
-for(let i = 0; i < cardDivArray.length; i++) {
+for (let i = 0; i < cardDivArray.length; i++) {
     cardDivArray[i].innerHTML = '<img class="hamster" src="' + shuffledCardsArray[i].frontOfCard + '">'
-        + '<img class="sloth" src="images/cardBack.jpg">';
+        + '<img class="sloth" data-pair ="' + shuffledCardsArray[i].pairID + '" src= "images/cardBack.jpg">';
+
 }
 
+let clickCounter = 0
+let turnCounter = 0
+let firstClickPair = "start"
+let secondClickPair = "start"
 let slothClickArray = document.querySelectorAll('.sloth');
```

Comment:
naming variables using "sloth" breaks the principle of separation of concern - if you decide to change the image on the back the name of the variable will be very confusing. change the name of the variable to something like `cardBacks`

Reply:
also change any other sloth related or hamster related names

Reply:
change them to frontOfCard and backOfCard please harry